show/main.py

```diff
@@ -1329,6 +1329,29 @@ def all(verbose):
     run_command(cmd, display_cmd=verbose)
 
 
+# 'accesslist' subcommand ("show runningconfiguration accesslist")
+@runningconfiguration.command()
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def accesslist(verbose):
```

Comment:
We use "acl" in our other "show" commands ("show acl table", "show acl rule", etc.). Should we rename this "show runningconfiguration acl" for consistency?

Reply:
Renamed it to acl. Done.